autopush-common/src/db/bigtable/bigtable_client/mod.rs

```diff
@@ -509,15 +509,35 @@ impl BigTableClientImpl {
         let resp = retry_policy(self.settings.retry_count)
             .retry_if(
                 || async {
-                    bigtable
+                    let resp: grpcio::ClientSStreamReceiver<bigtable::ReadRowsResponse> = bigtable
                         .conn
                         .read_rows_opt(&req, call_opts(self.metadata.clone()))
+                        .inspect_err(|e| {
+                            warn!("🉑 Read Rows failed: {:?}", &e);
+                        })?;
+                    merge::RowMerger::process_chunks(resp)
+                        .await
+                        .map_err(|e| match e {
+                            error::BigTableError::GRPC(e)
+                            | error::BigTableError::Read(e)
```

Comment:
I don't see these 2 variants being emitted in `merge` but I imagine you're guarding against them potentially happening in the future.

Being generic across `BigTableError` here convinced me we might as well clean all this up by making a `retryable_bigtable_error` version of `retryable_error`. How does [this](https://github.com/mozilla-services/autopush-rs/compare/SYNC-4508_retry...SYNC-4508_retry-pjenvey) look?